framework/src/main/java/org/checkerframework/common/value/util/Range.java

```diff
@@ -1184,4 +1185,27 @@ private static Range bigRangeToLongRange(BigInteger bigFrom, BigInteger bigTo) {
         long longTo = bigTo.longValue();
         return createRangeOrEverything(longFrom, longTo);
     }
+
+    /**
+     * Returns a Range representing all possible values for the given primitive type.
+     *
+     * @param typeKind is a java.lang.TypeKind that isPrimitive() and integral.
+     * @return the matching range for that primitive type.
+     */
+    public static Range byPrimitiveTypeKind(TypeKind typeKind) {
```

Comment:
This name doesn't reflect what the routine does.  Could you make it use some words from the first sentence of the documentation, or otherwise a bit more obvious?

Reply:
When read in context, the names of the new functionality refactor-extracted from them are not so foreign:
return Range.byPrimitiveTypeKind(resolveToPrimitiveIntegralTypeKind(type)).from;
and 
return Range.byPrimitiveTypeKind(resolveToPrimitiveIntegralTypeKind(type)).to;

Reply:
I am open to suggestions for a better name.